rush-plugins/rush-bridge-cache-plugin/src/BridgeCachePlugin.ts

```diff
@@ -122,6 +123,7 @@ export class BridgeCachePlugin implements IRushPlugin {
                   terminal.writeLine(
                     `Operation "${operation.name}": Outputs have been restored from the build cache."`
                   );
+                  terminal.writeLine(`Cache key: ${projectBuildCache.cacheId}`);
```

Comment:
Are these useful for devs? Don't they already end up in a log file?

Reply:
No, there's no log file for this action. And yes, it's pretty useful. When we launched our new build pipeline a couple of weeks ago, the inability to trace back the actual build artifacts from this populate Rush cache step got immediately called out.